### PR TITLE
PSR-1226 transfers-in CYA view only page

### PR DESCRIPTION
**Pull request by narendrapaduchuri**

### CODE REVIEW

File: app/controllers/nonsipp/receivetransfer/TransfersInCYAController.scala [hunk 1]
```diff
@@ -55,51 +58,82 @@ class TransfersInCYAController @Inject()(
...
-                )
-            )
-            .map {
-              case Nil => Redirect(controllers.routes.JourneyRecoveryController.onPageLoad())
-              case journeys => Ok(view(viewModel(srn, memberDetails.fullName, index, journeys, mode)))
-            }
-            .merge
-        }
-      ).merge
+      onPageLoadCommon(srn: Srn, index: Max300, mode: Mode)(implicitly)
```
Comment (kberec) [inline app/controllers/nonsipp/receivetransfer/TransfersInCYAController.scala:61]: Just spotted this, it was a mistake in the original example I think. Should this be: 
```
onPageLoadCommon(srn, index, mode)(implicitly)
```
? (same in onPageLoadViewOnly)

Comment (kberec) [reply app/controllers/nonsipp/receivetransfer/TransfersInCYAController.scala:61]: If you agree, could you please change it in basic / financial details too @narendrapaduchuri ?

**Review by kberec — DISMISSED** [review]

File: app/controllers/nonsipp/receivetransfer/TransfersInCYAController.scala (hunk 1 above)
Comment (narendrapaduchuri) [reply app/controllers/nonsipp/receivetransfer/TransfersInCYAController.scala:61]: Yes, @kberec I agree with you. I changed across above mentioned and  member contributions as well. Thanks

**Review by ccatosdevelopment — APPROVED** [review]

File: app/controllers/nonsipp/receivetransfer/TransfersInCYAController.scala (hunk 1 above)
Comment (kberec) [reply app/controllers/nonsipp/receivetransfer/TransfersInCYAController.scala]: Thanks. Just the commits needs squashed and it is good to get merged.

**Review by kberec — APPROVED** [review]

**Pull request merged**